src/Card.tsx

```diff
@@ -284,95 +402,211 @@ export default function Card({
+                  >
+                    <FontAwesomeIcon icon={faPerson}/>
+                  </button>
+                  <button 
+                    className="p-1 px-2 rounded-full transition-all"
+                    style={draftCardType == "repeater" ? {
+                      background: THEMES[theme].background,
+                      color: THEMES[theme].color
+                    } : {
+                      background: "none",
+                      color: THEMES[theme].background
+                    }}
+                    onClick={() => setDraftCardType("repeater")}
+                  >
+                    <FontAwesomeIcon icon={faTowerBroadcast}/>
+                  </button>
+                </div>
+              ) : (
+                contact.cardType == "person" ? (
+                  <FontAwesomeIcon icon={faPerson} className="text-4xl"/>
+                ) : (
+                  <FontAwesomeIcon icon={faTowerBroadcast} className="text-3xl"/>
+                )
+              )}        
+            </div>
+          </div>
+          <div className={editMode ? "w-64" : ""}>
+            {(editMode && draftCardType == "person") || (!editMode && contact.cardType == "person") ? (
+              <Input
+                className="text-3xl"
+                value={displayName}
+                onChange={(e) => setDraftName(e.target.value)}
+                onKeyDown={handleInputKeyDown}
+                placeholder="name"
+                disabled={!editMode}
+              />
+            ) : null} 
+            {(editMode && draftCardType == "repeater") || (!editMode && contact.cardType == "repeater") ? (
+              <Input
+                className="text-3xl"
+                value={displayLocation}
+                onChange={(e) => setDraftLocation(e.target.value)}
+                onKeyDown={handleInputKeyDown}
+                placeholder="location"
+                disabled={!editMode}
+              />     
+            ) : null}
+          </div>
         </div>
-        <div className="-mb-1 grid grid-cols-[2.1rem,1fr] z-10">
-          {dmr?.count ? (
-            <Field label="DMR">
-              <a
-                href={`https://radioid.net/database/view?id=${dmr.results[0].id}`}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {dmr.results[0].id}
-              </a>
-            </Field>
-          ) : null}
-          {(contact.website || editMode) && (
-            <Field label="WEB">
-              {createMode || editMode ? (
-                <div className="w-36">
+        {(editMode && draftCardType == "repeater") || (!editMode && contact.cardType == "repeater") ? (
+          <div className="z-10 text-sm w-52">
+            <div className="grid grid-cols-2 gap-2">
+              {editMode || contact.frequency ? (
+                <Input
+                  className=""
+                  value={displayFreq}
```

Comment:
doing the repeater fields in the monospace font might make the numbers look a bit nicer? with the slashed zero and all that. feel free to resolve if that isn't your style tho

Reply:
Oh yeah that does make sense